glusterd2/xlator/validation.go

```diff
@@ -52,6 +52,29 @@ func validateBitrot(v *volume.Volinfo, key string, value string) error {
 	return nil
 }
 
+func validateQuota(v *volume.Volinfo, key string, value string) error {
+
+	// Check if quota is already enabled
+	if volume.IsQuotaEnabled(v) == false {
+		err := fmt.Errorf("Quota not enabled to set this value: '%s'", key)
+		return err
+	}
+
```

Comment:
What exactly is this function for ? Are these values supposed to be allowed to change when quota is enabled or when it's disabled ? `validateQuota` is always called for any volume set option which has `quota` prefix in it.

Reply:
Yes. All these values can be set only if quota is enabled.

Reply:
> Yes. All these values can be set only if quota is enabled.

What about other quota keys ?

Reply:
The other quota keys like limit set, remove limit, etc.. will be added here when we work on those patches and send the PR.